lib/cubits/statistics/statistics_cubit.dart

```diff
@@ -10,10 +9,8 @@ part 'statistics_state.dart';
 
 class StatisticsCubit extends Cubit<StatisticsState> {
   final LeaderboardRepository leaderboardRepository;
-  final AccountRepository accountRepository;
 
-  StatisticsCubit(this.leaderboardRepository, this.accountRepository)
-      : super(StatisticsState());
+  StatisticsCubit(this.leaderboardRepository) : super(StatisticsState());
 
   void filterStatistics(StatisticsFilterCategory filterBy) {
```

Comment:
I don't understand this method. Can you explain what it does?

Reply:
sure.

1. if a refresh is triggered with the same filter, ignore
2. the dropdown has changed, change the cubit state to the new one
3. fetchLeaderboards() is called where it fetches data with the current selected preset (stored in the cubit)